src/ring_theory/witt_vector/mul_coeff.lean

```diff
@@ -0,0 +1,339 @@
+/-
+Copyright (c) 2022 Robert Y. Lewis. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Robert Y. Lewis, Heather Macbeth
+-/
+
+import ring_theory.witt_vector.truncated
+import data.mv_polynomial.supported
+
+/-!
+# Leading terms of Witt vector multiplication
+
+The goal of this file is to study the leading terms of the formula for the `n+1`st coefficient
+of a product of Witt vectors `x` and `y` over a ring of characteristic `p`.
+We aim to isolate the `n+1`st coefficients of `x` and `y`, and express the rest of the product
+in terms of a function of the lower coefficients.
+
+For most of this file we work with terms of type `mv_polynomial (fin 2 × ℕ) ℤ`.
+We will eventually evaluate them in `k`, but first we must take care of a calculation
+that needs to happen in characteristic 0.
+
+## Main declarations
+
+* `witt_vector.nth_mul_coeff`
```

Comment:
Can you add a descripion of the result? Or maybe modify the section `Main declarations`, I am not sure which one is better.

Reply:
```suggestion
* `witt_vector.nth_mul_coeff`: expresses the coefficient of a product of Witt vectors in terms of the previous coefficients of the multiplicands. 
```